invenio_records_rest/utils.py

```diff
@@ -40,6 +40,33 @@
     PIDUnregisteredRESTError
 
 
+@cached_property
```

Comment:
What about `default_endpoint_prefixes`? (since it's not actually a endpoint what is provided but just the prefix to be used to build the set of end points, `{0}_list`, `{0}_item`...)